### ISSUE TITLE
[Bug]: Mac installer issues

### ISSUE THREAD

**Issue by adamphetamine** [gravitl/netmaker#1813]
### Contact Details

adam.connor@servicemax.com.au

### What happened?

I upgraded the Server to Netmaker 0.17.0
Installed v0.17.0 of the Netclient-M1.pkg installer on a Mac with previously working netclient.
Machine did not show updated netclient in dashboard (still shows as v0.16.1). 

Your installer drops a receipt on the Mac that defines the package version and installed version as v0.16.2
Your Intel Mac installer has this issue as well.

Troubleshooting
In addition, running 'sudo netclient pull' on this Mac fails with the messages below in log paste.
Can ping the Netmaker server from Mac client, but cannot connect to other nodes

I verified that this seems to be an exclusively macOS issue by upgrading a Synology NAS to v0.17.0 and this worked without issue, and 'netclient pull' also ran without issue.

Also as mentioned before, please don't deposit a locked folder in the /Applications folder to store secrets. If you want to obfuscate stuff, put it in /usr or even better in /Library/Application Support/netclient

### Version

v0.17.0

### What OS are you using?

Mac

### Relevant log output

```shell
MacStudio:~$ sudo netclient pull
[netclient] 2022-12-11 13:14:49 No network selected. Running Pull for all networks. 
[netclient] 2022-12-11 13:15:08 error running command: wg-quick up /Applications/Netclient/config/nm-1.conf 
[netclient] 2022-12-11 13:15:08 [+] Interface for nm-1 is utun0
wg-quick: `nm-1' already exists as `utun0' 
[netclient] 2022-12-11 13:15:08 network: net2 UDP hole punching enabled for node localnode
[netclient] 2022-12-11 13:15:11 error running command: wg-quick up /Applications/Netclient/config/nm-2.conf 
[netclient] 2022-12-11 13:15:11 [+] Interface for nm-2 is utun2
wg-quick: `nm-2' already exists as `utun2' 
[netclient] 2022-12-11 13:15:31 error running command: wg-quick up /Applications/Netclient/config/nm-3.conf 
[netclient] 2022-12-11 13:15:31 [+] Interface for nm-3 is utun3
wg-quick: `nm-3' already exists as `utun3'
```

### Contributing guidelines

- [X] Yes, I did.

**Comment by jakoberpf** [comment]
I also have this issue. @adamphetamine did you find a workaround?

```logs
#  ~ sudo netclient pull -n test
[netclient] 2023-02-07 19:16:39 network: test UDP hole punching enabled for node Jakobs-MBP-2fritzbox 
[netclient] 2023-02-07 19:16:42 error running command: wg-quick up /Applications/Netclient/config/nm-test.conf 
[netclient] 2023-02-07 19:16:42 [+] Interface for nm-test is utun7
wg-quick: `nm-test' already exists as `utun7'
```

**Comment by adamphetamine** [comment]
@jakoberpf I had to drop this after 9 months of working on the Mac version, sorry I can't really tell what the error message means- I'd post it in the Discord channel and someone will help there...

**Comment by piotr-sikora-v** [comment]
I have fresh install nectlient on my mac and fresh install Netmaker server
Have a similar error:
```
xx@Xxxx ~ > sudo netclient pull -n test-net
[netclient] 2023-02-16 10:48:39 network: test-net node Xxxxlocal is using port 51821 
[netclient] 2023-02-16 10:48:41 error running command: wg-quick up /Applications/Netclient/config/nm-test-net.conf 
[netclient] 2023-02-16 10:48:41 [+] Interface for nm-test-net is utun7
wg-quick: `nm-test-net' already exists as `utun7'
```

strange is that I can connect and I see only my Netmaker server

BTW... I have installed wireguard gui client, byt deactivated and killed.

**Comment by adamphetamine** [comment]
Sorry @piotr-sikora-v, 
don't know what's happening here, but you can see from the logs that the error is in wg-quick. 
As far as I can recall, that's a small bash script that was supposed to be integrated into the app, but when testing my own installer it still seemed to be required, and built as a separate executable. <<I'd look at this first 
You could make your own installer using the code I published but it's a lot of work...

**Comment by abhishek9686** [comment]
Check out the Mac version of Netclient from the latest releases. Significant updates have been implemented since v0.17.0.
